eks/cluster-loader/cluster-loader.gotmpl

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cl-service
+automountServiceAccountToken: true
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    name: cl-service
+    namespace: default
+
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cl2
+spec:
+  template:
+    spec:
+      containers:
+      - name: clusterloader2
+        image: 197575167141.dkr.ecr.us-west-2.amazonaws.com/clusterloader2:latest
```

Comment:
It looked like you were trying to make this image configurable with your eksconfig, but it's hardcoded here.

Reply:
Ah yes, good catch, let me fix.